airbyte-ci/connectors/cdk/cdk/java_cdk/ci.py

```diff
@@ -0,0 +1,81 @@
+
+from typing import Awaitable, List, Optional
+
+from aircmd.models.base import PipelineContext
+from aircmd.models.click_commands import ClickCommandMetadata, ClickFlag, ClickGroup
+from aircmd.models.click_params import ParameterType
+from aircmd.models.utils import make_pass_decorator
+from dagger import Client, Container
+from prefect import flow
+from prefect.utilities.annotations import quote
+
+from .settings import JavaCDKSettings
+from .tasks import build_java_cdk_task, test_java_cdk_task
+
+pass_pipeline_context = make_pass_decorator(PipelineContext, ensure=True)
+pass_global_settings = make_pass_decorator(JavaCDKSettings, ensure=True)
+
+java_group = ClickGroup(group_name="java", group_help="Commands for developing Airbyte Java CDK")
+
+
+class CICommand(ClickCommandMetadata):
```

Comment:
Curious what you think about having the classes like this. One nice thing you can do is add pydantic validators in here to validate arguments, like if you wanted to ensure that some string you passed in was valid. It does separate the command from the actual flow though, which may not be desirable.